bench-plotter/src/bench_plotter/plotting/query_utils.py

```
@@ -0,0 +1,186 @@
+        instant_query_func: Function to execute instant queries
+
+    Returns:
+        First successful payload or the last attempted payload
+    """
+
+    def run_query(qs: str) -> Dict[str, Any]:
+        try:
+            return asyncio.run(
+                query_range_func(query=qs, start_unix=start_time, end_unix=end_time)
+            )
+        except Exception as e:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)